Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JoiSchemaEnhancer.ts

```diff
@@ -237,6 +244,20 @@ function joiArrayFor(property: EntityProperty, propertyModifier: PropertyModifie
   );
 }
 
+/**
+ * Returns a Joi schema fragment that specifies the API body element shape
+ * corresponding to the given school year enumeration.
+ */
+function joiTypeForSchoolYearEnumeration(property: EntityProperty, propertyModifier: PropertyModifier): Joi.AnySchema {
+  // We ignore whether this is a collection because there are no collections of school year in the DS,
+  // so the shape of a school year collection is unknown.
+  const result = Joi.object().keys({
```

Comment:
Just to highlight this point - not sure what a collection would look like, apart from whether it would make sense to ever model one for school years.